gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -530,7 +531,7 @@ public void start() throws ApplicationException {
     this.orchestrator.getSpecCompiler().setActive(true);
 
     //Activate the DagManager service, after the topologyCatalog has been initialized.
```

Comment:
please add a comment explaining we don't want to start DM threads (e.g. as mentioned in the PR desc)